### PR TITLE
chore: add more info for subscription item in env tree view

### PR DESCRIPTION
**Pull request by dooriya**
Fix bug: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/12607395
- Add subscription id info as the tree item description for each subscription node.
- Add tooltip for subscription node (user can mouse over to see more info about this node)

image

### CODE REVIEW

**Comment by qinezh** [comment]
Looks it's hardly to display the full subscription id without resizing the tree view, how about putting the subscription id to the tooltip instead?

**Comment by dooriya** [comment]
> Looks it's hardly to display the full subscription id without resizing the tree view, how about putting the subscription id to the tooltip instead?

Sounds good. In the latest commit, I also added subscription id info in the tooltip.

**Review by qinezh — DISMISSED** [review]

File: packages/vscode-extension/src/envTree.ts [hunk 1]
```diff
@@ -255,11 +255,23 @@ async function appendSubscriptionAndResourceGroupNode(env: string): Promise<void
...
+
     if (subscriptionInfo) {
       const subscriptionTreeItem: TreeItem = {
         commandId: `fx-extension.environment.subscription.${env}`,
         contextValue: "openSubscriptionInPortal",
         label: subscriptionInfo.subscriptionName ?? subscriptionInfo.subscriptionId,
+        description: subscriptionInfo.subscriptionId,
+        tooltip: {
+          isMarkdown: false,
+          value: `'${env}' environment is provisioned in Azure subscription '${
```
Comment (HuihuiWu-Microsoft) [inline packages/vscode-extension/src/envTree.ts:267]: Can you extract the string to Strings.json for later localization?

**Review by a1exwang — DISMISSED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

File: packages/vscode-extension/src/envTree.ts (hunk 1 above)
Comment (dooriya) [reply packages/vscode-extension/src/envTree.ts]: Sure. Thanks for suggestion.

**Pull request merged**